app/models/parent.rb

```diff
@@ -1,4 +1,5 @@
 class Parent < ApplicationRecord
+  has_many :rewards
```

Comment:
Think we want to add a 'dependent: destroy' after the has_many :rewards

Reply:
updated with latest commit!